.github/workflows/quality_control.yml

```diff
@@ -0,0 +1,35 @@
+# This a workflow to ensure all written project code passes pylint and pytest
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**.py'
+  workflow_dispatch:
+jobs:
+  quality_control:
+    name: "Quality Control"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout the Code"
+        uses: actions/checkout@v4
+      - name: "Install Python"
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+      - name: "Install all libraries"
+        run: |
+          pip3 install -r requirements.txt
+          echo "library installation complete"
+      - name: "Run Pytest"
+        run: |
+            pytest --only-path="**.py"
+            echo "Ran pytest"
+      - name: "Run Pylint"
+        run: |
+            pylint --fail-under=8 *.py
```

Comment:
I believe this line should be `$(find . -name "*.py")`

Reply:
Changed this line